### DISCUSSION TITLE
Testing: `refresh` not working

### DISCUSSION THREAD

**Discussion by amshehzad** [livewire/livewire#7842] [7. Bugs (archived)] [answered]
### Livewire version

3.4.4

### Laravel version

v10.42.0

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

`Livewire::test(ExampleComponent::class)->refresh();`
this throws error:

 Method Illuminate\Http\Response::refresh does not exist.

### Repository or Wirebox URL

-

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@amshehzad try this instead `Livewire::test(ExampleComponent::class)->call('$refresh');`

**Comment by amshehzad** [comment]
@joshhanley 
Yeah I'm doing this currently or `Livewire::test(ExampleComponent::class)->commit();` but just reporting a bug as `refresh` is mentioned in the docs but I can see that it's not in the `Testable` class
https://livewire.laravel.com/docs/testing#setup-methods

**Reply by joshhanley** [reply]
@amshehzad ah right sorry! You didn't mention it was in the docs but not working. Yeah that makes sense, it should work if it is documented, PRs to fix are welcome 🙂

**Comment by amshehzad** [comment, marked as answer]
Opened PR here [#7844 ](https://github.com/livewire/livewire/pull/7844)

**Reply by joshhanley** [reply]
@amshehzad awesome, thanks! 🤩

**Reply by calebporzio** [reply]
Merged!
